gem/oq-engine
# Made the download of hazard curves fast

And stopped saving the individual hazard curves from the GMFs, only the statistics are saved, consistently with what is done in the classical calculator.

Closes https://github.com/gem/oq-engine/issues/3899.

The differences in the expected files can be understood as follows:

1. when there is a single realization, we export it as mean, so the header is different from before
2. because of a 64bit -> 32bit conversion the hazard curves generated by the GMFs are slightly different then before
3. openquake/qa_tests_data/ucerf/expected/hmap.rst looks different since I have changed the ordering in the view (this is used only for debugging, it is not a real output)
4. the numbers in the files in openquake/qa_tests_data/event_based/case_7/expected look different because before I was looking at the individual realizations, now at the quantiles, but they are actually the same; idem for case_6
5. before I was generating two files openquake/qa_tests_data/event_based/case_2/expected/hc-smltp_b1-gsimltp_b1-ltr_0.csv and  openquake/qa_tests_data/event_based/case_2/expected/hc-smltp_b1-gsimltp_b1-ltr_1.csv, now I am generating only the mean file. 

To test it:

Download with QGIS the outputs of https://oq2.wilson.openquake.org/engine/18744/outputs: this is a
large calculation with 126,044 sites, but you will see that the startup time before the real download starts is of only a few seconds. The download time will be dominated by your connection speed, as it should be.

The QGIS tests are green: https://travis-ci.org/gem/oq-irmt-qgis/builds/416827960
